DATA/production/configurations/asyncReco/async_pass.sh

```diff
@@ -586,10 +586,15 @@ else
     echo "WORKFLOW_PARAMETERS=$WORKFLOW_PARAMETERS"
     echo "Step 3) matching, calib, AOD, potentially QC"
     echo -e "\nStep 3) matching, calib, AOD, potentially QC" >> workflowconfig.log
-    export TIMEFRAME_RATE_LIMIT=${ALIEN_JDL_TIMEFRAMERATELIMITSSPLITWF:-0}
+    # This uses the same time frame rate limiting as in full wf, unless differently specified in the JDL
+    export TIMEFRAME_RATE_LIMIT=${ALIEN_JDL_TIMEFRAMERATELIMITSSPLITWF:-${TIMEFRAME_RATE_LIMIT}}
```

Comment:
is TIMEFRAME_RATE_LIMIT guaranteed to be defined upstream?

Reply:
In principle, it should always be there. If not, then it will just be zero, the jobs will crash, and we will fix it.